ci/autoscaler/tasks/update-sdk/update_java_package.sh

```diff
@@ -48,12 +36,13 @@ if [ "${JAVA_VERSION}" == "${current_java_version}" ]; then
   exit 0
 fi
 
-# Step 1 --> Download java...
-source "${script_dir}/download_java.sh" "${JAVA_VERSION}"
```

Comment:
This means we can remove `download_java.sh`, correct?

Reply:
Correct... Done.. Thanks